charts/map/templates/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "map.fullname" . }}
+  labels:
```

Comment:
Need to add kubemonkey labels like here - https://github.com/GaloyMoney/charts/pull/5473